packager/packager.cc

```diff
@@ -1028,13 +1048,36 @@ struct PackageSpecBodyWalk {
                     return;
                 }
                 info.visibleToTests_ = true;
+            } else if (auto target = ast::cast_tree<ast::Send>(send.getPosArg(0))) {
+                // Constant::* is valid Ruby, and parses as a send of the method * to Constant
+                // so let's take advantage of this to implement wildcards
+                if (target->fun != core::Names::star()) {
+                    if (auto e = ctx.beginError(target->loc, core::errors::Packager::InvalidConfiguration)) {
+                        e.setHeader("Argument to `{}` must be a constant or the string literal `{}`",
+                                    send.fun.show(ctx), "\"tests\"");
+                    }
+                    return;
+                }
+                if (auto recv = verifyConstant(ctx, send.fun, target->recv)) {
+                    auto importArg = move(target->recv);
+                    send.removePosArg(0);
+                    ENFORCE(send.numPosArgs() == 0);
+                    send.addPosArg(prependName(move(importArg)));
+                    info.visibleTo_.emplace_back(getPackageName(ctx, recv), true);
+                } else {
+                    if (auto e = ctx.beginError(target->loc, core::errors::Packager::InvalidConfiguration)) {
+                        e.setHeader("Argument to `{}` must be a constant or the string literal `{}`",
+                                    send.fun.show(ctx), "\"tests\"");
+                    }
+                    return;
+                }
             } else if (auto *target = verifyConstant(ctx, send.fun, send.getPosArg(0))) {
                 auto importArg = move(send.getPosArg(0));
                 send.removePosArg(0);
                 ENFORCE(send.numPosArgs() == 0);
                 send.addPosArg(prependName(move(importArg)));
 
-                info.visibleTo_.emplace_back(getPackageName(ctx, target));
+                info.visibleTo_.emplace_back(getPackageName(ctx, target), false);
```

Comment:
Can you describe this `false` (and the `true` above in the other half of the case), either with a comment like `/* isWildCard */` or by assigning it to a variable, like `auto isWildcard = false`?